spatialmath/mesh.go

```diff
@@ -539,3 +544,56 @@ func calculatePolygonAreaWithTriangulation(vertices []r3.Vector) float64 {
 		return totalArea
 	}
 }
+
+// trianglesToPLYBytes converts the mesh's triangles to bytes in PLY format.
+func (m *Mesh) trianglesToPLYBytes() []byte {
+	// Collect all unique vertices and create vertex-to-index mapping
+	vertexMap := make(map[string]int)
+	vertices := make([]r3.Vector, 0)
+
+	for _, tri := range m.triangles {
+		for _, pt := range tri.Points() {
+			scaledPt := r3.Vector{X: pt.X / 1000.0, Y: pt.Y / 1000.0, Z: pt.Z / 1000.0}
```

Comment:
hmmm.
so we are converting mm to m?

why convert at all?

The unit is dependent on the the pointcloud coordinate system from which the mesh was built.

Reply:
Is it not standard that PLY files are denominated in meters? Certainly we assume they are in the existing code where we  turn a ply back to a mesh